sos/report/plugins/pulp.py

```diff
@@ -160,8 +160,8 @@ def postproc(self):
         # Now handle JSON-formatted data in the same /etc/pulp directory
         # structure. We use a different substitution string here to preserve
         # the file's JSON syntax.
-        jreg = r"(\s*\".*(passw|cred|token|secret).*\"\s*:\s*\")(.*)(\")"
-        repl = r"\1********\4"
+        jreg = r"(\"proxy_password\"(\s)*:(\s))(.*)(\w+)"
+        repl = r"\1********"
```

Comment:
This removes obfuscation of anything aside from `proxy_password`, which is not an acceptable change.

What needs to be investigated is why `proxy_password` is not being matched by the current regex (`.*passw.*`, which should be matching it).

Reply:
Hm.. maybe just different quotation mark? The current `jreg` uses `"`, my example in #2564 has `'`.

Maybe (writing from head):

            jreg = r"(\s*\".*(passw|cred|token|secret).*\"\s*:\s*(\w+)"

will cover anything after `:`, hence just enhancing the regexp to cover also my example?